aggregator/pom.xml

```diff
@@ -71,6 +71,28 @@
             <version>${spark-rapids-private.version}</version>
             <classifier>${spark.version.classifier}</classifier>
         </dependency>
+        <!--
+            placeholders for maximum number of delta-lake shims per Spark shim:
+            - if we need only N use rapids.delta.artifactIdN+1 = rapids.delta.artifactIdN
+        -->
```

Comment:
Why is this refactoring part of an already large PR to add a single shim?  This belongs in a separate PR.

Reply:
Because it is the use case that drives it. However, I see your point for making it separate